pkg/gds/client/profiles_test.go

```diff
@@ -0,0 +1,196 @@
+package client_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/shibukawa/configdir"
+	"github.com/stretchr/testify/require"
+	"github.com/trisacrypto/directory/pkg/gds/client"
+	"gopkg.in/yaml.v2"
+)
+
+const profileYAML = "profiles.yaml"
+
+// Test that DefaultProfiles returns a Profiles object with some valid profiles.
+func TestDefaultProfiles(t *testing.T) {
+	profiles := client.DefaultProfiles()
+	require.NotNil(t, profiles)
+	require.True(t, strings.HasPrefix(profiles.Version, "v"))
+	require.NotEmpty(t, profiles.Profiles)
+	require.Contains(t, profiles.Profiles, profiles.Active)
```

Comment:
Good generic tests that will make it easy to maintain the `DefaultProfiles` function!